simple_interest.c

```diff
@@ -0,0 +1,22 @@
+/* Find the simple interest if the principal amount is Rs. 2000, the time period is 2 years, 
+and the rate is 10%. Also, calculate the total amount at the end of 1 year*/
+
+#include <stdio.h>
+int main(void)
+{
+    int p,n,A;
+    float r,si;
+    printf("Enter p:Rs.");
+    scanf("%d",&p);
+    printf("Enter n:");
+    scanf("%d",&n);
+    printf("Enter r:");
+    scanf("%f",&r);
```

Comment:
Could you please use only a single pritf() and scanf() function for p, n and r

Reply:
Sir, I made the required changes and pre-assigned the values for p, n, and r.